src/pages/community/foundation/local-initiative.mdx

```diff
@@ -0,0 +1,136 @@
+---
+sidebarTitle: GraphQL Local
+description:
+  The GraphQL Local initiative helps the GraphQL Foundation foster a culture of
+  community and innovation, contributing to the success, health, and
+  sustainability of our ecosystem.
+---
+
+import { Button } from "../../../app/conf/_components/button"
+
+# GraphQL Local Initiative
+
+The GraphQL Foundation's primary mission is to ensure the continued evolution of
+the GraphQL specification and reference implementations. As part of this
+mission, we strive to provide collaboration infrastructure that fosters a
+culture of community and innovation. Local get-togethers play a crucial role in
+this, bringing together developers, library maintainers, vendors, and
+enthusiasts to share knowledge and collaborate.
+
+To better support these meets and clarify the process, we are excited to
+announce the official **GraphQL Local** initiative. This initiative provides a
+framework for organizing local GraphQL user groups and offers support each approved
+group (hereafter, a "Local"). Here’s what you need to know to get involved:
+
+## Requirements
+
+Organizers of a Local must agree:
+
+- All official events and related channels will be governed by the
+  **[GraphQL Code of Conduct](https://graphql.org/codeofconduct/)**.
+- Official communications will comply with the
+  **[GraphQL Brand Guidelines](https://graphql.org/brand/)**.
+- The Local accepts **GraphQL TSC oversight**:
+  - At least one
+    [GraphQL Technical Steering Committee (TSC) member](https://github.com/graphql/graphql-wg/blob/main/GraphQL-TSC.md#tsc-members)
+    must have administrator access to the Local’s online platforms.
+  - In case of a serious Code of Conduct breach, an organizer may be replaced.
+- The Local has 1 or more organizer from the area. (We're pretty flexible on the
+  definition of "from the area", just be reasonable.)
+- Each official event will have at least least one **organizer present**.
+- Official events will be held in an appropriate **professional setting**.
```

Comment:
Should we clarify with some examples? E.g. would a pub qualify, or a games arcade?

Reply:
I'm happy to leave this open, if they have questions/venue in mind then they can ask. For example, I think the main room of a pub is probably not an appropriate setting, but exclusive use of the events room in a pub is probably fine.